bundles/org.openhab.binding.fronius/README.md

```diff
@@ -1,54 +1,89 @@
+| `inverterdatachanneludc`             | Number    | DC voltage                                                                                                        |
+| `inverterdatadevicestatuserrorcode`  | Number    | Device error code                                                                                                 |
+| `inverterdatadevicestatusstatuscode` | Number    | Device status code<br />`0` - `6` Startup<br />`7` Running <br />`8` Standby<br />`9` Bootloading<br />`10` Error |
+| `powerflowchannelpgrid`              | Number    | Power + from grid, - to grid                                                                                      |
+| `powerflowchannelpload`              | Number    | Power + generator, - consumer                                                                                     |
+| `powerflowchannelpakku`              | Number    | Power + charge, - discharge                                                                                       |
+
+### Channels for `meter` Thing
+
+| Channel ID              | Item Type | Description                                                                                                                                                                                                              |
+| ----------------------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `model`                 | String    | The model name of the meter                                                                                                                                                                                              |
+| `serial`                | String    | The serial number of the meter                                                                                                                                                                                           |
+| `enable`                | Number    | 1 = enabled, 0 = disabled                                                                                                                                                                                                |
+| `location`              | Number    | 0 = grid interconnection point (primary meter)<br/> 1 = load (primary meter)   <br />3 = external generator (secondary meters)(multiple)<br />256-511 = subloads (secondary meters)(unique). Refer to Fronius Solar API. |
+| `currentacphase1`       | Number    | AC Current on Phase 1                                                                                                                                                                                                    |
```

Comment:
Can you use Units of Measure? See https://www.openhab.org/docs/concepts/units-of-measurement.html
```suggestion
| `currentacphase1`       | Number:ElectricCurrent    | AC Current on Phase 1                                                                                                                                                                                                    |
```

Reply:
done